rules/kotlin/kotlin.bzl

```diff
@@ -1,5 +1,46 @@
 load("@io_bazel_rules_kotlin//kotlin:jvm.bzl", _kt_jvm_library = "kt_jvm_library")
 load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", _kt_compiler_plugin = "kt_compiler_plugin")
+load("@grab_bazel_common//rules/android/lint:defs.bzl", "lint", "lint_sources")
+load("@grab_bazel_common//rules/android/lint:providers.bzl", "LINT_ENABLED")
 
 kt_jvm_library = _kt_jvm_library
 kt_compiler_plugin = _kt_compiler_plugin
+
+def kt_jvm_library_interal(
+        name,
+        exec_properties = None,
+        lint_options = {},
+        **attrs):
+    lint_sources_target = "_" + name + "_lint_sources"
+    lint_baseline = lint_options.get("baseline", None)
+
+    lint_sources(
+        name = lint_sources_target,
+        srcs = attrs.get("srcs"),
+        baseline = lint_baseline,
+        lint_config = lint_options.get("lint_config", None),
+    )
+
+    kt_jvm_library(
+        name = name,
+        srcs = attrs.get("srcs"),
+        tags = attrs.get("tags", default = []) + [LINT_ENABLED],
```

Comment:
Can we use `**attrs`?

Similar to https://github.com/grab/grab-bazel-common/blob/7c9717930a9c2cbfd270324a68001fc1227155a8/rules/kotlin/kotlin.bzl#L27

Reply:
we will get duplicated arg because we need to modify tags and deps